packages/js/product-editor/src/components/custom-fields/create-modal/create-modal.tsx

```diff
@@ -238,6 +241,15 @@ export function CreateModal( {
 										'key'
 									) }
 									onBlur={ blurHandler( customField, 'key' ) }
+									className={ classNames(
+										'woocommerce-product-text-control',
```

Comment:
I don't think we're using this class. If we resolve to keep it, I would rename it since it's not related to the TextControl anymore.

Reply:
[e15a530](https://github.com/woocommerce/woocommerce/pull/48949/commits/e15a530cabd2d526104bb8ca286f75b545792cff)